### PR TITLE
2nd try for updating ntfs-3g to current stable release

### PR DESCRIPTION
**Pull request by z-e-r-o**
Patch is not needed because it is included into configure.ac of new stable version.

### CODE REVIEW

File: packages/sysutils/ntfs-3g_ntfsprogs/package.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-################################################################################
+7################################################################################
```
Comment (CvH) [inline packages/sysutils/ntfs-3g_ntfsprogs/package.mk]: `7` again :)

Comment (z-e-r-o) [inline packages/sysutils/ntfs-3g_ntfsprogs/package.mk]: What the hell... I thought I understood the system... and then it strikes again against me. :-)
Nevertheless the one provided by you is correct.

**Comment by vpeter4** [comment]
Why exactly was the patch removed? Because it is not included in current ntfs-3g version.

Because of this missing patch package doesn't build for imx6 project with 3.14 kernel. Not really important atm because kodi also doesn't build. But maybe this will changed in next half year.

**Comment by vpeter4** [comment]
Sorry for the noise but patch is really not needed. Problem in my case was that kodi was not build and also not libgcrypt. Which is needed for ntfs3g.

All is good.

**Pull request merged**